contracts/NodeIngress.sol

```diff
@@ -22,29 +22,18 @@ contract NodeIngress is Ingress {
     }
 
     function connectionAllowed(
-        bytes32 sourceEnodeHigh,
-        bytes32 sourceEnodeLow,
-        bytes16 sourceEnodeIp,
-        uint16 sourceEnodePort,
-        bytes32 destinationEnodeHigh,
-        bytes32 destinationEnodeLow,
-        bytes16 destinationEnodeIp,
-        uint16 destinationEnodePort
-    ) public view returns (bytes32) {
+        string memory sourceEnodeId,
```

Comment:
The `source` prefix doesn't make sense anymore. We should remove it from the parameter names. I think we should use `enodeId`, `ip` and `port`.